package.json

```diff
@@ -35,7 +35,7 @@
     "express": "^4.13.4",
     "express-session": "^1.13.0",
     "html-entities": "^1.2.0",
-    "html-to-react": "^0.1.2",
+    "html-to-react": "^1.0.0",
```

Comment:
The issue was that the library did not support void html tags, such as `<br>` and would generate warnings if they appeared in a component template. Support was added in their 1.0.0 release.